pkg/ccl/changefeedccl/sink_webhook.go

```diff
@@ -187,12 +208,75 @@ func makeWebhookClient(u sinkURL, timeout time.Duration) (*httputil.Client, erro
 	return client, nil
 }
 
-// Dial is a no-op for this sink since we don't necessarily have
-// a "health check" endpoint to use.
+type retryConfig struct {
+	backoff    time.Duration
+	maxRetries uint
+}
+
+func defaultRetryConfig() retryConfig {
+	return retryConfig{
+		backoff:    500 * time.Millisecond,
```

Comment:
open to discussing this default config further (similar to kafka except backoff 250 -> 500)